README.md

```diff
@@ -0,0 +1,49 @@
+# animate
+Anisotropic mesh adaptation toolkit for Firedrake
+
+
+    The implementation of metric-based mesh adaptation used in PETSc assumes that the
+    metric is piece-wise linear and continuous, with its degrees of freedom at the
+    mesh vertices.
+
+Animate is an mesh adaptation toolkit enabling users to anisotropically adapt a mesh based on a Riemannian metric framework with control over the shape, orientation and size of the resulting mesh elements. Animate is built for use with Firedrake and  strongly leverages PETSc DMPlex functionality. The implementation of metric-based mesh adaptation used in PETSc assumes that the metric is piece-wise linear and continuous, with its degrees of freedom at the mesh vertices.
+
+For more information on Firedrake, please see: [Firedrake Website](https://www.firedrakeproject.org/).
+For more information on PETSc, please see: [PETSc metric based mesh adaptation](https://petsc.org/release/docs/manual/dmplex/#metric-based-mesh-adaptation)
+
+## Linux Installation
+
+The following installation instructions assume a linux or WSL operating system. The options below include installation from a custom shell script which also installs the custom setup for Firedrake and PETSc. If Firedrake is already installed, please see instructions to install Animate via git clone.
```

Comment:
`git clone`, i.e. use code styling.

Reply:
I've done this in both the title and the body of the text